README.md

```diff
@@ -26,7 +26,8 @@ Kubernetes v1.12+.
 In this setup, the directory `/opt/local-path-provisioner` will be used across all the nodes as the path for provisioning (a.k.a, store the persistent volume data). The provisioner will be installed in `local-path-storage` namespace by default.
 
 ```
-kubectl apply -f https://raw.githubusercontent.com/rancher/local-path-provisioner/master/deploy/local-path-storage.yaml
+kuebctl apply -f deploy/local-path-storage.yaml
```

Comment:
Hmm, we cannot do this. This will stop the one-line installation of Local Path Provisioner.

Better have the script in the config map by default.

Reply:
i removed cm dir and put the scripts into deploy/local-path-storage.yam with a script configmap by default.